### DISCUSSION TITLE
Reserve static IP when creating VPC endpoint

### DISCUSSION THREAD

**Discussion by cfabriziohivemq** [upbound/provider-aws#1388] [Q&A]
We cannot assign or reserve a static IP when creating a VPC Endpoint and associating the subnets via `VPCEndpointSubnetAssociation`. This functionality is referred to as a "designated IP" when selecting the subnet through the console.

**Comment by mergenci** [comment]
@cfabriziohivemq, you “cannot” because “it's not possible” or because “it didn't work”? It would help us if you could provide example manifests. If what you are trying to achieve is available in Terrafrom, could you provide a link to the resource?
